packages/requisitions/src/RequestRequisition/DetailView/AppBarButtons/UseSuggestedQuantityButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  ButtonWithIcon,
+  EditIcon,
+  useTranslation,
+  useConfirmationModal,
+} from '@openmsupply-client/common';
+import { useSuggestedQuantity } from '../../api';
+
+export const UseSuggestedQuantityButtonComponent = () => {
+  const t = useTranslation('replenishment');
+  const { mutate: setRequestedToSuggested } = useSuggestedQuantity();
+  const getConfirmation = useConfirmationModal({
+    onConfirm: setRequestedToSuggested,
+    message: t('messages.requested-to-suggested'),
+    title: t('heading.requested-to-suggested'),
+  });
+
+  return (
+    <ButtonWithIcon
+      Icon={<EditIcon />}
```

Comment:
at one point Manu has a lightning bolt icon for the 'wizard' actions.. that might be nice here. 
leave it for now, I can add that in later and see how it looks

Reply:
Oh! That's right. I did wonder if I should use the popover similarly to how you've done the `Print` with these extra actions. The wizard modal was cool though.

For this one (and the response one), I'd imagine the API should change to accept lines and for this to become an action in the drop down.

Maybe another button like mobile: "One click order", which populates the requisition with all of the items which have a suggested quantity > 0 and sets the requested to the suggested